### PR TITLE
Merge xmos_v1.20.1 into master

### PR DESCRIPTION
**Pull request by lucianomartin**
Please note: some of the changes are already on master.

Summary of changes:
   - update AVS device SDK to version v1.20.1 (already merged on master)
   - add support for configurable Sensory operating point
   - add support for XMOS AVS tests by running the avsrun command with the option XMOS_AVS_TESTS

### CODE REVIEW

File: tools/Install/setup.sh [hunk 1]
```diff
@@ -138,9 +138,9 @@ fi
 
 XMOS_TAG=$2
 
-shift 1
+shift 2
 
-OPTIONS=s:a:m:d:h:x
+OPTIONS=s:a:m:d:hx:
```
Comment (mbanth) [inline tools/Install/setup.sh:143]: Did you intend to join the h and x options without an intervening colon?

**Review by mbanth — COMMENTED** [review]
One quick question.

File: tools/Install/setup.sh (hunk 1 above)
Comment (lucianomartin) [reply tools/Install/setup.sh:143]: The column was wrong in the original code. It should be added only when we are reading an extra CLI argument.
In this case '-h' doesn't have any extra argument, while '-x' does.

Comment (mbanth) [reply tools/Install/setup.sh:143]: Thanks for the clarification.

**Review by mbanth — APPROVED** [review]
All looks good.

**Pull request merged**